packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -214,23 +214,23 @@ export class EpochTracker {
                 // This will only throw if it is an epoch error.
                 this.checkForEpochErrorCore(epochFromResponse, error.errorMessage);
             } catch (epochError) {
-                const err = {
-                    ...epochError,
+                assert(epochError instanceof LoggingError);
```

Comment:
Add a small message please :)

Reply:
Removing static debug strings was proven to be # 1 way to reduce binaries in devmain. Our bundles are not different. There is no need to sprinkle messages all over the place, it's similar to commenting - they are useful here and there, but if you have to comment every line of code, that likely means code can be improved to be more readable.
I'd rather see Tony's work completing and inserting tags all over place, and messages (converted to comments) being optional.